io.github.nest_desktop.nest-desktop.yml

```
@@ -0,0 +1,84 @@
+id: io.github.nest_desktop.nest-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+  - org.freedesktop.Sdk.Extension.typescript
+command: nest-desktop-launch
+separate-locales: false
+rename-icon: nest-desktop
+rename-desktop-file: nest-desktop.desktop
+finish-args:
+  # https://github.com/endlessm/electron-installer-flatpak?tab=readme-ov-file#optionsfinishargs
+  # X Rendering
+  - --socket=x11
+  - --share=ipc
+  # Open GL
+  - --device=dri
+  # Audio output
+  - --socket=pulseaudio
+  # Read/write home directory access
+  - --filesystem=home
+  # Chromium uses a socket in tmp for its singleton check
+  - --filesystem=/tmp
+  # Allow communication with network
+  - --share=network
+  # System notifications with libnotify
+  - --system-talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: nest-desktop
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/nest-desktop/nest-desktop
+        branch: dev
+        commit: 6acecc1923605f2d4cb8d95c5ed49697b6fd1f98
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+          - exec zypak-wrapper.sh /app/main/nest-desktop "$@"
+      - type: file
+        path: io.github.nest_desktop.nest-desktop.metainfo.xml
+    build-options:
+      build-args:
+        - --share=network
+      append-path: /usr/lib/sdk/node20/bin
+      env:
+        NPM_CONFIG_LOGLEVEL: info
+    build-commands:
+      # Install yarn and remove package-lock.json
+      - |
+        npm install yarn
+        rm package-lock.json
+      # Install node modules
+      - yarn install
+      # Build the electron app
+      - yarn app:build
+      # Bundle app and dependencies
+      - mv release/linux-unpacked /app/main
+      # Install app wrapper
+      - install -Dm0755 run.sh "${FLATPAK_DEST}/bin/nest-desktop"
+      - install -Dm0755 scripts/nest-desktop-launch "${FLATPAK_DEST}/bin/nest-desktop-launch"
+      - install -Dm644 icons/icon-256x256.png /app/share/icons/hicolor/256x256/apps/nest-desktop.png
+      - install -Dm644 icons/icon-512x512.png /app/share/icons/hicolor/512x512/apps/nest-desktop.png
+      - install -Dm644 io.github.nest_desktop.nest-desktop.metainfo.xml ${FLATPAK_DEST}/share/metainfo/nest-desktop.metainfo.xml
+      - install -Dm0755 nest-desktop.desktop "${FLATPAK_DEST}/share/applications/nest-desktop.desktop"
+
+  - name: nest-simulator
+    buildsystem: cmake
```

```suggestion
    buildsystem: cmake-ninja
```

Unless there is a good reason not too